src/App.js

```diff
@@ -4,6 +4,7 @@ import { Container } from "./Components/Container.styled";
 import { Body } from "./Components/Body.styled";
 
 import WeatherLoader from "./Components/WeatherLoader";
+import { Input } from "./Components/input.styled";
```

Comment:
as all your component names commence with uppercase letters, please adhere to the same naming convention.